server/src/main/resources/db/common/V82__drop_feedback_request_questions_table.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS feedback_request_questions;
```

Comment:
are we just dropping this table ? and in feedback branch I only see migration files upto V78 so why did we name this V82 ?

Reply:
We are dropping this table because we no longer need it in our current schema, but I didn't want to mess with the other migrations. The file was named V82 to try to minimize conflict, but I will rename it.